styles.css

```diff
@@ -0,0 +1,103 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+body {
+  font-size: 1rem;
+  font-weight: 300;
+  line-height: 1.6;
+  color: white;
+}
+
+body::before {
+  content: "";
+  position: fixed;
+  top: 0;
+  left: 0;
+  height: 100%;
+  width: 100%;
+  z-index: -2;
+  background: #1b1b32;
+  background-image: linear-gradient(
+      100deg,
+      rgba(50, 50, 150, 0.5),
+      rgba(150, 150, 200, 0.5)
+    ),
+    url(http://photographyblogger.net/wp-content/uploads/2016/09/rsz_surfing_7.jpg);
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+h1 {
+  font-weight: 300;
+  line-height: 1.6;
+}
+p {
+  font-size: 1.3rem;
+}
+h1,
+p {
+  margin-top: 0;
+  margin-bottom: 0.6rem;
+}
+label {
+  display: flex;
+  align-items: center;
+  font-size: 1.3rem;
+  margin-bottom: 0.5rem;
+}
+
+.container {
+  max-width: 720px;
+  margin: 3rem auto 0 auto;
+}
+
+form {
+  background-color: rgba(27, 27, 50, 0.8);
+  padding: 3rem 1rem;
+  border-radius: 1.5rem;
+}
+
+.form-group {
+  margin: 0 auto 0.5rem auto;
+  padding: 0.25rem;
+}
+.form-control {
+  display: block;
+  width: 100%;
+  /* height: 0rem; */
+  padding: 0.5rem 1rem;
```

Comment:
You should not comment unwanted codes because it makes your code unclean.

Reply:
Good job 👍 but You should ask them to remove it.